recipes/pymbar/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "pymbar" %}
+{% set version = "3.0.2" %}
+{% set sha256 = "6a161ea32addddc26384b2e32086b28996bd1e7b0d759472452e86a6c4cf90dc" %}
+
+package:
+  name: {{ name  }}
+  version: {{ version }}
+
+source:
+  fn: {{ name}}-{{ version }}.tar.gz
+  url: https://github.com/choderalab/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy
+    - scipy
+    - six
+    - numexpr
+  run:
+    - python
+    - numpy
+    - scipy
+    - six
+    - numexpr
+
+test:
+  requires:
+    - nose
+    - statsmodels
+  imports:
+    - pymbar
+  commands:
+    - nosetests pymbar --with-doctest --doctest-tests -v
+
+about:
+  home: http://github.com/choderalab/pymbar
+  license: MIT
+  license_file: LICENSE
+  summary: 'Python implementation of the multistate Bennett acceptance ratio (MBAR) method.'
+
+extra:
+  recipe-maintainers:
+    - Lnaden
+    - jchodera
```

Comment:
No problem. I always get excited when I see projects from your group or any of the other MD projects show up on conda-forge.